.github/workflows/build.yml

```diff
@@ -11,11 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        idf_ver: ["release-v4.1", "release-v4.2", "release-v4.3"]
+        idf_ver: ["release-v4.3"]
```

Comment:
makes sense to add some newer versions? 4.4, 5.0 and 5.1

Reply:
They are already there in `build-examples-pedantic` job below, around line 29.

As I understand, when we have enabled more strict warnings checks in CI, we only did so for v4.4 and later, and split the jobs into
- a job for the old releases without warnings checks
- a job for the new releases with warnings checks

I guess the two jobs could be folded into one. But it's probably not so important since release/v4.3 will be EOL in a month (December 15) and we can remove this job completely.

Reply:
ah... that explains my confusion.